src/sentry/models/grouphash.py

```diff
@@ -38,6 +39,7 @@ class State:
     state = BoundedPositiveIntegerField(
         choices=[(State.LOCKED_IN_MIGRATION, _("Locked (Migration in Progress)"))], null=True
     )
+    date_added = models.DateTimeField(default=timezone.now, null=True)
```

Comment:
The default we're adding here ends up causing us to backfill all the rows with the current date. So what we should do here is either:
 - Switch this to be a `db_default` and make the column not nullable
 - Modify the migration so that we don't set the default on the column for existing rows. You'd do something like (rough code)
 
```
SeparateDatabaseAndState(
db_operations=[
        migrations.AddField(
            model_name="grouphash",
            name="date_added",
            field=models.DateTimeField(null=True),
        ),],
state_operations=[
        migrations.AddField(
            model_name="grouphash",
            name="date_added",
            field=models.DateTimeField(default=django.utils.timezone.now, null=True),
        ),
])
```